bin/buildSVUnit

```diff
@@ -54,6 +56,8 @@ sub usage() {
   print "  -m|--mock                : includes the uvm_mock_pkg to the final build file\n";
   print "  -U|--uvm                 : build SVUnit with UVM\n";
   print "  -w|--wavedrom            : process json files as wavedrom output\n";
+  print "     --relative-paths      : use relative paths instead of absolute paths\n";
+  print "  -i|--incdir              : use -i for incdir instead of standard +incdir+\n";
```

Comment:
I don't think we need this at all. We can hide the fact that Vivado needs `-i`, while the others use `+incdir+`. This way, the user just needs to supply `--simulator xsim`.

Reply:
buildSVUnit does not have a simulator option. Should I pass that to buildSVUnit instead?